doc/classes/Parallax2D.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="Parallax2D" inherits="Node2D" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		A node used to create a parallax scrolling background.
+	</brief_description>
+	<description>
+		A [Parallax2D] is used to create a parallax effect. It can move at a different speed relative to the camera movement using [member scroll_scale]. This creates an illusion of depth in a 2D game. If manual scrolling is desired, the [Camera2D] position can be ignored with [member ignore_camera_scroll].
+		[b]Note:[/b] Any changes to this node's position made after it enters the scene will be ignored.
+	</description>
+	<tutorials>
+	</tutorials>
+	<members>
+		<member name="autoscroll" type="Vector2" setter="set_autoscroll" getter="get_autoscroll" default="Vector2(0, 0)">
+			Velocity at which the offset scrolls automatically, in pixels per second.
+		</member>
+		<member name="follow_viewport" type="bool" setter="set_follow_viewport" getter="get_follow_viewport" default="true">
+			If [code]true[/code], this [Parallax2D] is offset by the current camera's position. It may be desired to disable it if the [Parallax2D] is in a [CanvasLayer] separate from the current camera.
+		</member>
+		<member name="ignore_camera_scroll" type="bool" setter="set_ignore_camera_scroll" getter="is_ignore_camera_scroll" default="false">
+			If [code]true[/code], [Parallax2D]'s position is not affected by the position of the camera.
+		</member>
+		<member name="limit_begin" type="Vector2" setter="set_limit_begin" getter="get_limit_begin" default="Vector2(0, 0)">
+			Top-left limits for scrolling to begin. If the camera is outside of this limit, the [Parallax2D] stops scrolling. Must be lower than [member limit_end] minus the viewport size to work.
+		</member>
+		<member name="limit_end" type="Vector2" setter="set_limit_end" getter="get_limit_end" default="Vector2(0, 0)">
+			Bottom-right limits for scrolling to end. If the camera is outside of this limit, the [Parallax2D] will stop scrolling. Must be higher than [member limit_begin] and the viewport size combined to work.
+		</member>
+		<member name="offset" type="Vector2" setter="set_offset" getter="get_offset" default="Vector2(0, 0)">
+			The [Parallax2D]'s offset relative to its parent.
```

Comment:
Yeah, it's tricky since the position is overridden, and there's no way around that. `offset` gives the benefit of offsetting while still respecting the infinite scroll, rather than just moving the whole effect off screen, which probably wouldn't be desirable in any case.

I could disable setting `position` in the editor, but if they managed to, they'd have no way to fix it... and messing with it doesn't break anything since it just gets overridden anyway. I could just set whatever `position` they have in the editor to be the `offset` when the scene starts, but that might be confusing since there'd be no way to reset it once the scene starts... so I'm open to ideas.

Reply:
You could actually have a warning when setting the `position`, but it's a bit tricky and it's not a deal breaker at all. It can be added later **if** this is merged.

Reply:
I was on the fence about adding a configuration warning, because I wasn't sure if there was a valid scenario for position to ever be changed, but I think you're right. It's probably safer to add it for now. Also, I think it'd be better to name this `scroll_offset` since it scrolls through the infinite repeat effect rather than just offsetting the position.

Reply:
I'm not against it. There's a lot of offsets being used here in Parallax2D, so being more specific does not hurt. Reminds me of https://github.com/godotengine/godot/pull/64804